api/swagger.yml

```diff
@@ -59,17 +59,21 @@ externalDocs:
   description: Find out more about Hermez network.
   url: 'https://hermez.io'
 servers:
-  - description: Hosted mock up
-    url: https://apimock.hermez.network/v1
-  - description: Localhost mock Up
-    url: http://localhost:4010/v1
+  - description: Hosted mock up, returns fake data usefull for development
+    url: https://apimock.hermez.network
+  - description: Localhost mock up, returns fake data usefull for development
+    url: http://localhost:4010
+  - description: Testnet (Rinkeby) server
+    url: https://api.testnet.hermez.io
+  - description: Mainnet (Ethereum) server, use it carefully, specially if attempting to send transactions. You could lose money!
+    url: https://api.hermez.io
```

Comment:
I'm not sure if this information should go here. I understand this swagger config file as the file to run the mock api locally, but not as the directory of the current available deployed servers.
I think that `Hosted mockup`, `Testnet server` and `Mainnet server` should not go to this file, and should go to the documentation website.

Reply:
the idea is that the UI is actually a nice playground of the API, and I thought that it was a good idea to let people play with real data

Reply:
I'm not too fond of the idea of putting these config URLs into the swagger file. It's easy to lost track and forget to update. We can create a script to deploy it with a dynamic URL passing by parameter for someone who wants to use it as a playground. But IHMO is we must keep this swagger file only for we can debug the deployed node.